linera-views/src/scylla_db.rs

```diff
@@ -184,7 +184,12 @@ impl ReadableKeyValueStore<ScyllaDbContextError> for ScyllaDbStoreInternal {
 
 #[async_trait]
 impl DirectWritableKeyValueStore<ScyllaDbContextError> for ScyllaDbStoreInternal {
-    const MAX_BATCH_SIZE: usize = usize::MAX;
+    // The constant 14000 is an empirical constant that was found to be necessary
+    // to make the ScyllaDb system work. We have not been able to find this or
+    // a similar constant in the source code or the documentation.
+    // An experimental approach gets us that 14796 is the latest value that is
+    // correct.
+    const MAX_BATCH_SIZE: usize = 12000;
```

Comment:
I would recommend a much lower value like `2000` or `5000`. I doubt that this will be a performance bottleneck (but you're welcome to prove me wrong with data).

Reply:
Ok.